Integrations/integration-VMRay.yml

```diff
@@ -74,8 +74,7 @@ script:
         # and stores it internaly. To get the filename you need an automation.
         # You can use demisto.args()[argName] to get a specific arg. args are strings.
         file = demisto.args()['file_id']
-        path = demisto.getFilePath(file)
```

Comment:
I think this a bad merge that overrides a fix. @meirwah right?

Reply:
true, @Spivaka plz fix this merge

Reply:
accidental, fixing..